PROJECT

```diff
@@ -11,8 +11,8 @@ resources:
     namespaced: true
   controller: true
   domain: ovn.org
-  group: self.service
+  group: k8s
```

Comment:
I am unsure about this as well.
Unless we are placed under the OVN umbrella, hopefully as an independent project, I do not think you can set this domain and group to follow it.

But I may be wrong, I suggest to consult with the OVN org maintainers/leaders.

Reply:
One of the options for productizion this project is to migrate it ovn.org under ovn-k repo, which is the preferred option, therfore I change the API grpup/domain to fit other ovn-k APIs.

Lets revisit this topic once we get feedback from ovn-k community about the APIs, and whether it"ll  be accommodated in ovn-k repo or not.

Reply:
Once we get the OVN community approval we can revisit this indeed. Until then, I suggest not to touch it.

Reply:
Done